src/pages/example--hello-world.mdx

```diff
@@ -289,7 +306,7 @@ If you notice the small icon next to the contract output address, we can even [i
 }
 ```
 
-### Unlocking funds from the contract
+### Unlocking Funds From The Contract
```

Comment:
There's not much of a reason to capitalize the first letter of every word in titles.

Reply:
right